Gems/PhysX/Code/Editor/ColliderComponentMode.cpp

```diff
@@ -96,6 +82,17 @@ namespace PhysX
             SetCurrentMode(SubMode::Rotation);
         });
 
+        AzToolsFramework::ActionOverride setDimensionsModeAction;
+        setDimensionsModeAction.SetUri(SetDimensionsSubModeActionUri);
+        setDimensionsModeAction.SetKeySequence(QKeySequence(Qt::Key_3));
+        setDimensionsModeAction.SetTitle("Set Resize Mode");
+        setDimensionsModeAction.SetTip("Set resize mode");
+        setDimensionsModeAction.SetEntityComponentIdPair(GetEntityComponentIdPair());
+        setDimensionsModeAction.SetCallback([this]()
+            {
+                SetCurrentMode(SubMode::Dimensions);
+            });
```

Comment:
uber nit: I don't mind which indentation we use but can this be the same as the lambdas above please? 😝

Reply:
Ran clang format on the whole file.